### ISSUE TITLE
Conflicting Schema.org labels

### ISSUE THREAD

**Issue by eeeps** [c2pa-org/specifications#43]
In the [Use of Schema.org](https://c2pa.org/specifications/specifications/1.4/specs/C2PA_Specification.html#_use_of_schema_org) section, the spec states,

> All Schema.org assertions shall have a label that starts with `schema.org` and be followed by the name of the schema that is being used. For example, `schema.org.ImageObject` can be used to include something extra about an image.

This somewhat conflicts with this statement in the intro to [Labels](https://c2pa.org/specifications/specifications/1.4/specs/C2PA_Specification.html#_labels)

> Well-established standards can use the "stds." prefix when describing their namespace.

(EXIF and IPTC clearly/consistently use the `stds.` prefix).

It also conflicts with the [Standard C2PA Assertion Summary](https://c2pa.org/specifications/specifications/1.4/specs/C2PA_Specification.html#_standard_c2pa_assertion_summary)...

<blockquote>
<table>
</tr><tr>
<td>Claim Review
<td>stds.schema-org.ClaimReview
<td>Schema.org ClaimReview
<td>JSON-LD
<tr>
<td>Creative Work
<td>stds.schema-org.CreativeWork
<td>Schema.org CreativeWork
<td>JSON-LD
</tr>
</table>
</blockquote>

...the [Creative Work section](https://c2pa.org/specifications/specifications/1.4/specs/C2PA_Specification.html#_creative_work )...

> A Creative Work assertion shall have a label of `stds.schema-org.CreativeWork`.

...and many examples in the spec. A find in page for "stds.schema-org." returns 21 results, all for labels of `stds.schema-org.CreativeWork` or `stds.schema-org.ClaimReview`. A find-in-page for "schema.org." returns five relevant results, all for `claim_generator.claim_generator_info[].schema.org.SoftwareApplication.operatingSystem`.

Most importantly, existing tooling (Verify and c2patool), which does not yet implement the Common Metadata system introduced in 1.4, only reads from `stds.schema-org.CreativeWork` assertions, not `schema.org.CreativeWork`.

Not sure what the best path forward here is (especially given depreciation); perhaps an informational note about `stds.schema-org.CreativeWork` and `stds.schema-org.ClaimReview` could be added to the "Use of Schema.org" introduction, for as long as that section still exists in the spec?

**Comment by lrosenthol** [comment]
@eeeps You are absolutely correct.  Most of this is, however, already corrected in the 1.4 version of the specification, and future releases will also improve it even further.

**Comment by eeeps** [comment]
@lrosenthol The links and text above are all from the current public 1.4 version of the specification.

By "corrected", do you mean that Schema.org labels have been marked as deprecated in 1.4, and so these conflicts are not important to resolve because these labels won't be present in the spec at all, soon? I am inclined to agree, except that current tooling does not yet implement Common Metadata, so implementors (like me) might find themselves consulting these deprecated sections to figure out what to do in the meantime.

**Comment by lrosenthol** [comment]
This issue has been migrated to the c2pa-org/specs-core repository. Please continue the discussion at https://github.com/c2pa-org/specs-core/issues/2176
